planetary-social/nos
# Remove info button from Discover tab

## Issues covered
#1094
#671, to some degree

## Description
Removed the info button from the Discover tab. Also disabled tests that fail intermittently, as those are the worst kind of tests; keeping them around will just train us to ignore failing tests.

## How to test
1. Navigate to Discover
2. Observe that there's no info button

## Screenshots/Video
| Before | After |
|--------|--------|
| ![Simulator Screenshot - iPhone 15 Pro - 2024-05-01 at 11 41 08](https://github.com/planetary-social/nos/assets/59564/aafaead8-6cd0-4a70-93a9-dce89911455c) | ![Simulator Screenshot - iPhone 15 Pro - 2024-05-01 at 11 40 14](https://github.com/planetary-social/nos/assets/59564/7d3fe56d-f754-46c3-bc1b-ef06c3d07813) |
